test/unit/helpers.bats

```diff
@@ -257,4 +259,22 @@ load _helpers
       yq '.spec.initContainers[] | select(.name == "get-auto-encrypt-client-ca").volumeMounts[] | select(.name=="consul-ca-cert")' | tee /dev/stderr)
 
   [ "${actual}" = "" ]
-}
\ No newline at end of file
+}
+
+#--------------------------------------------------------------------
+# bootstrapACLs deprecation
+#
+# Test that every place global.bootstrapACLs is used, global.acls.manageSystemACLs
+# is also used.
+# If this test is failing, you've used either only global.bootstrapACLs or
+# only global.acls.manageSystemACLs instead of using the backwards compatible:
+#     or global.acls.manageSystemACLs global.bootstrapACLs
+@test "helper/bootstrapACLs: used alongside manageSystemACLs" {
+  cd `chart_dir`
+
+  diff=$(diff <(grep -r '\.Values\.global\.bootstrapACLs' templates/*) <(grep -r '\.Values\.global\.acls\.manageSystemACLs \.Values\.global\.bootstrapACLs' templates/*) | tee /dev/stderr)
```

Comment:
This is a great way to test this out! As extra insurance that we're using things correctly, would it make sense to make sure that these two values are associated with an `or`?

So, using `grep -r 'or \.Values\.global\.acls\.manageSystemACLs \.Values\.global\.bootstrapACLs' templates/*` instead. There are two places that would need an order change for this to work:

```
< templates/client-clusterrole.yaml:{{- if (or .Values.global.enablePodSecurityPolicies .Values.global.acls.manageSystemACLs .Values.global.bootstrapACLs) }}

< templates/client-snapshot-agent-clusterrole.yaml:{{- if (or .Values.global.enablePodSecurityPolicies .Values.global.acls.manageSystemACLs .Values.global.bootstrapACLs) }}
```

Reply:
Yeah that's safer. Done!